lib/jekyll-github-metadata/repository.rb

```diff
@@ -98,6 +98,10 @@ def releases
         memoize_value :@releases, Value.new(proc { |c| c.releases(nwo) })
       end
 
+      def commits
+        memoize_value :@commits, Value.new(proc { |c| c.commits(nwo) })
```

Comment:
Okay I see what'll need to happen, but unfortunately I'm not proficient in Ruby. How would I go about implementing this?